APE24.rst

```diff
@@ -0,0 +1,443 @@
+reviewer for Affiliated request, namely:
+
+* familiarity with the Astropy project,
+* ability to judge whether a package integrates well with the Astropy ecosystem
+  (as per pre-APE 24 guidelines), and
+* having domain expertise in the area of the package (e.g., galaxy evolution).
+
+pyOpenSci and Astropy both ask editors/reviewers disclose any potential conflict of interest (COI)
+prior to agreeing to review a package. In the event where COI occurs, with this partnership,
+we would follow the
+`pyOpenSci COI process <https://www.pyopensci.org/software-peer-review/our-process/policies.html#conflict-of-interest-for-reviews-and-editors>`_.
+pyOpenSci invites the Astropy community to review this language to ensure it meets our needs.
+
+.. _reviewer sign-up form: https://forms.gle/GHfxvmS47nQFDcBM6
+
+Packaging guidelines
+--------------------
+
+pyOpenSci has been developing a `community-driven packaging guide`_
+that covers modern best practices and recommendations for scientific Python packaging.
+This guide has a stringent community review process. Reviewers for this guide
+represent members of PyPA, core Python, Anaconda (conda/conda-forge), and
+core packages for front- and back-end tools (e.g., ``flit``, PDM, ``hatch``/``hatchling``, etc.).
+It is also maintained by `Scientific Python`_.
+
+Meanwhile, the Astropy community prefers a more laid-back
+`OpenAstronomy packaging guide <https://packaging-guide.openastronomy.org/en/latest/>`_.
+
+If we partner with pyOpenSci, we would encourage our Affiliated packages to follow
+their `community-driven packaging guide`_ so that they are consistent with the
+broader scientific community, allowing the streamlining of new contributor experience
+across different domains.
+
+Regardless, the
+`Editor-in-Chief checks <https://www.pyopensci.org/software-peer-review/how-to/editor-in-chief-guide.html#editor-checklist-template>`_
+are the bare minimum for a package being able to go through the pyOpenSci review process.
+
+.. _community-driven packaging guide: https://www.pyopensci.org/python-package-guide/
+
+Listing on websites
+-------------------
+
+The pre-APE 24 `Astropy Affiliated Packages`_ listing are listed by alphabetical order
+with hard-coded badges. While these badges are color-coded to give a at-a-glance status
+of packages, they become outdated over time and now could even be misleading
+(i.e., they really are showing the status at the time of acceptance, not the current status).
+pyOpenSci is willing to work with Astropy and the broader scientific community to develop
+a more consistent standard of evaluating the "health state" of a package, using
+universally accepted metrics for such measurements (see **"Re-review" of packages**);
+therefore, getting rid of the need for static badges altogether.
+
+pyOpenSci is willing to create the following specifically for Astropy
+if we agree to this partnership:
+
+* A RSS/XML feed of Affiliated packages that we could ingest and
```

Comment:
do we want to specify the format here? we could do xml! but i already also have a yaml file that you could grab right now. so i just wonder rather than specifying the format, if we want to just say a automated way for astropy to access package information and publish on their website. something a bit more generic. then we can talk about what format makes the most sense.

Reply:
Sure. I think I got that from one of those notes we took.